PurchasesTests/Purchasing/PurchasesOrchestratorTests.swift

```diff
@@ -0,0 +1,72 @@
+    var systemInfo: MockSystemInfo!
+    var subscriberAttributesManager: MockSubscriberAttributesManager!
+    var operationDispatcher: MockOperationDispatcher!
+    var receiptFetcher: MockReceiptFetcher!
+    var purchaserInfoManager: MockPurchaserInfoManager!
+    var backend: MockBackend!
+    var identityManager: MockIdentityManager!
+    var receiptParser: MockReceiptParser!
+    var deviceCache: MockDeviceCache!
+
+    var orchestrator: PurchasesOrchestrator!
+
+    override func setUp() {
+
+
+        productsManager = MockProductsManager()
+        storeKitWrapper = MockStoreKitWrapper()
+        systemInfo = try! MockSystemInfo(platformFlavor: "xyz",
+                                         platformFlavorVersion: "1.2.3",
+                                         finishTransactions: true)
+        operationDispatcher = MockOperationDispatcher()
+        receiptFetcher = MockReceiptFetcher(requestFetcher: MockRequestFetcher())
+        deviceCache = MockDeviceCache()
+        backend = MockBackend()
+        purchaserInfoManager = MockPurchaserInfoManager(operationDispatcher: OperationDispatcher(),
+                                                        deviceCache: deviceCache,
+                                                        backend: backend,
+                                                        systemInfo: systemInfo)
+        identityManager = MockIdentityManager(mockAppUserID: "appUserID")
+        receiptParser = MockReceiptParser()
+        let attributionFetcher = MockAttributionFetcher(attributionFactory: MockAttributionTypeFactory(),
+                                                        systemInfo: systemInfo)
+        subscriberAttributesManager = MockSubscriberAttributesManager(backend: backend,
+                                                                      deviceCache: deviceCache,
+                                                                      attributionFetcher: attributionFetcher,
+                                                                      attributionDataMigrator: MockAttributionDataMigrator())
+
+        orchestrator = PurchasesOrchestrator(productsManager: productsManager,
+                                             storeKitWrapper: storeKitWrapper,
+                                             systemInfo: systemInfo,
+                                             subscriberAttributesManager: subscriberAttributesManager,
+                                             operationDispatcher: operationDispatcher,
+                                             receiptFetcher: receiptFetcher,
+                                             purchaserInfoManager: purchaserInfoManager,
+                                             backend: backend,
+                                             identityManager: identityManager,
+                                             receiptParser: receiptParser,
+                                             deviceCache: deviceCache)
+    }
```

Comment:
This file is not really testing anything right?

Reply:
not yet. I think I lost some changes 😂 not bad to have as a base. but yeah, not testing anything right now. I think I was waiting for the separate schemes PR, but then forgot

Reply:
honestly hard to recall at this point

Reply:
@vegaro this isn't testing anything yet, but my thinking was to add tests for this file as a separate PR now, since this got big enough already and it unlocks being able to test in a more reasonable way